lib/types.ts

```diff
@@ -1,6 +1,7 @@
-import builder = require('xmlbuilder');
-
-export const enum EnumChangefreq {
+import builder from 'xmlbuilder';
+// can't be const enum if we use babel to compile
```

Comment:
Babel is mostly for the use of jest as a test engine. Real compilation is done by tsc.